modules/admin/ref-rbac-permission-policies.adoc

```diff
@@ -3,6 +3,32 @@
 
 Permission policies in {product} are a set of rules to govern access to resources or functionalities. These policies state the authorization level that is granted to users based on their roles. The permission policies are implemented to maintain security and confidentiality within a given environment.
 
+You can define the following types of permissions in {product-short}:
+
+* resource type
+* basic
+
+The distinction between the two permission types depend on whether a permission includes a defined resource type.
+
+You can define the resource type permission using either the associated resource type or the permission name as shown in the following example:
+.Example resource type permission definition
```

Comment:
We need to give a space in between to make this example label appear in the next line. 

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/94683525/506ce9c4-d46f-41cb-a1ae-2b0386f0190a)